src/c_wrapper.jl

```diff
@@ -370,7 +370,14 @@ function _unsafe_scs_solve(model::_ScsDataWrapper{S,T}) where {S,T}
         pointer(model.c),
     )
     for (key, value) in model.options
-        setproperty!(model.settings, key, value)
+        if value isa AbstractString
+            value isa String ||
```

Comment:
Please don't use this short-circuiting, prefer the expanded form. It means that coverage isn't accurate, and we don't end up testing the `|| other` case.

Reply:
sure, if this is discouraged, then maybe it should be added as option to formatting? see e.g. https://domluna.github.io/JuliaFormatter.jl/stable/api/#JuliaFormatter.is_standalone_shortcircuit-Tuple{CSTParser.EXPR}

Reply:
Looks like there's a function, but it's not exposed as a setting: https://domluna.github.io/JuliaFormatter.jl/stable/api/#JuliaFormatter.format_text-Tuple{AbstractString}